content/_partials/toptoolbar.html.haml

```diff
@@ -128,6 +128,8 @@
             Conduct
           %a.dropdown-item{:href => expand_link('artwork')}
             Artwork
+          %a.dropdown-item{:href => expand_link('books')}
+            Jenkins Books
```

Comment:
@krisstern should this one also be just "Books"?

Reply:
Sure, done